src/sagetasks/nextflowtower/utils.py

```diff
@@ -0,0 +1,145 @@
+from datetime import datetime
+
+from sagetasks.nextflowtower.client import TowerClient
+from sagetasks.utils import dedup, update_dict
```

Comment:
Is there a reason you need to do this here?  Is it just ordering of the functions in this module?